server/data/visitSchedulerApiClient.ts

```diff
@@ -137,6 +137,15 @@ class VisitSchedulerApiClient {
     })
   }
 
+  startAmendVisit(visitReference: string): Promise<Visit> {
+    return this.restclient.put({
+      path: `/visits/${visitReference}`,
+      data: <UpdateVisitRequestDto>{
+        visitStatus: 'RESERVED',
+      },
+    })
+  }
+
```

Comment:
Only required to update the status when doing this, current update function sends everything, could maybe changed to update status only?

Reply:
Just wondering about the `RESERVED` here - as I think you've mentioned if amend is started and then abandoned for whatever reason the visit might automatically housekept! 

If we're re-using the same visit, it feels like there should perhaps be another status, e.g. `AMENDING` or something - this could signify what's going on; stop it simultaneously being amended by someone else; and, cron job could revert any it finds to `BOOKED` were they to be abandoned 🤔 ? Obviously would need some much wider discussion!

Reply:
@tpmcgowan yep, the code that calls this is currently commented out for the reason you say 🙂